Jenkinsfile

```diff
@@ -207,7 +207,10 @@ ansiColor('xterm') {
                 string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')
               ]) {
                 try {
-                  sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' > $HOME/.npmrc'
+                  // Copy & update config file
+                  sh 'cp .npmrc $HOME/.npmrc'
```

Comment:
Did we want to backup the existing npmrc in the home directory if it exists since other teams might be relying on it?

Reply:
The web team confirmed that they weren't. I'd rather have each team be responsible for setting up and tearing down the environments properly than perpetuate dependence on files that may not be there.

Reply:
Sound good to me. LGTM!